src/tasks/package.js

```diff
@@ -29,32 +29,33 @@ const packages = [
 module.exports = (config) => {
   json('package.json')
     .merge({
+      main: 'dist/cjs.js',
+      files: [
+        'dist',
+      ],
       engines: {
         // Some versions are skipped because of known issues, see https://github.com/webpack-contrib/organization/issues/7
         node: `>= ${config.minNode} < 5.0.0 || >= 5.10`,
       },
       scripts: {
-        start: 'yarn run serve:dev src',
-        build: "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
-        'clean:dist': 'del-cli dist',
-        lint: 'eslint --cache src test',
+        start: 'yarn run build -- -w',
+        build: 'cross-env NODE_ENV=production babel src -d dist',
+        'clean': 'del-cli dist test/builds',
+        lint: 'eslint --cache src',
         'lint-staged': 'lint-staged',
-        prebuild: 'yarn run clean:dist',
+        prebuild: 'yarn run clean',
         prepublish: 'yarn run build',
         release: 'yarn run standard-version',
         security: 'nsp check',
-        'serve:dev': 'nodemon $2 --exec babel-node',
-        test: 'jest',
+        develop: 'nodemon $2 --exec babel-node',
+        pretest: 'yarn run build && node test/webpack.build.js',
```

Comment:
So this is where I go back to perhaps running a unit & integration suite or trying to find a better way to utilize Jest in the context of Webpack ( a jest preset / plugin perhaps ).

One of Jest's biggest selling points is it's development experience. i.e...
- No compile step.
- Tests run on code changes only and only the tests that are related to that code change.
- Editor plugins for Jest execute in the background without needing to run a watch command via the command line.

While it's obviously more work, I think the end result for both maintainers & contributors would make it worth the effort.

I'll do a bit of research and see what it would take to test through Webpack without sacrificing all the best parts of Jest.

//cc @bebraw - Given your zeal for Jest, i'm sure you have an opinion or perhaps some insight here.